Lib/ldap/syncrepl.py

```diff
@@ -535,3 +536,71 @@ def syncrepl_refreshdone(self):
         follows.
         """
         pass
+
+
+class OpenLDAPSyncreplCookie:
+    """
+    OpenLDAPSyncreplCookie - allows a consumer to track a cookie across a
+    refreshAndPersist syncrepl session against a multi-provider OpenLDAP cluster
+    """
+
+    rid: int = 0
+    sid: int = 0
+    _csnset: Dict[int, str]
+
+    def __init__(self, cookie: AnyStr = "") -> None:
+        self._csnset = {}
```

Comment:
It's really minor, but I think it's inconsistent.
You set `self._csnset` default in `__init__` but for `self.rid` and `self.sid` - you don't. Something like this should be nicer, IMO

```
    def __init__(self, cookie: AnyStr = "") -> None:
        self.rid = 0
        self.sid = 0
        self._csnset = {}

Reply:
Ir was really a minor concern, and as you explained that there is no need to track the `rid` and `sid,` I'm totally okay with the way it is now.